index.html

```diff
@@ -7,7 +7,98 @@
 </head>
 
 <body>
-    <h1 class="main__title">lesson_1</h1>
+    <header class="header">
+        <h1 class="header__target" style="visibility: hidden">Магазин мебели</h1>
```

Comment:
```suggestion
        <h1 class="header__title visually-hidden">Магазин мебели</h1>
```
Заголовки лучше называть своими именами
Инлайновые стили лучше не использовать вообще, считается плохой практикой
Добавил класс visually-hidden, чуть позже добавим это в общий файл со стилями и будем использовать этот класс по необходимости